drivers/start_date_dq/app/views/start_date_dq/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,39 @@
+- content_for :title, @title
+%h1= @title
+= render '/warehouse_reports/breadcrumbs', tab: 'data-quality'
+%p View differences between the client's self-reported start date (DateToStreetESSH) and the actual entry date.
```

Comment:
```suggestion
%p View differences between the client's self-reported start date (DateToStreetESSH) and the entry date.
```
The two dates are _actually_ different dates, this just exposes how long the client said they were homeless prior to finding shelter.